plugins/aea-ledger-solana/setup.py

```diff
@@ -59,6 +59,7 @@
         "Operating System :: Unix",
         "Programming Language :: Python :: 3.9",
```

Comment:
What about 3.8 here?

Reply:
`anchorpy>=0.17.0` does not have support for python<3.9